### ISSUE TITLE
usability issues with 'hod batch'

### ISSUE THREAD

**Issue by boegel** [hpcugent/hanythingondemand#113]
* specified script should be checked for existence before submitting the HOD cluster
* absolute path to specified script should be resolved before submission
* script should be made executable automagically, to avoid:
```
/bin/sh: /user/scratch/gent/vsc400/vsc40023/hod/wordcount.sh: Permission denied
```

**Comment by boegel** [comment]
see also #115

**Comment by boegel** [comment]
fixed in #116
